Cql/Cql.Packaging/LibraryPackager.cs

```diff
@@ -31,12 +31,17 @@ public class LibraryPackager
     {
         internal LibraryPackager()
         {
-            TypeConverter = FhirTypeConverter.Create(ModelInfo.ModelInspector);
+            TypeConverter = FhirTypeConverter.Create(ModelInfo.ModelInspector, 0);
         }
 
-        internal LibraryPackager(TypeConverter? typeConverter)
+        internal LibraryPackager(int cacheSize)
```

Comment:
I think these new constructors move the responsibility of setting the cache size from the TypeConverter (where they belong) into the LibraryPackager. Ultimately, we just need the constructor that takes a TypeConverter. Especially the overload taking a converter + a cache size is confusing, since it allows either the passed in typeconverter or the passed in cache size to be relevant, but for the "outside user" it is unclear what happens what you pass both.  I think we should clean this up if possible.

Reply:
Upon further review, the cache size is only needed during the run in a context of a patient, not packaging or any other location. The user can control the cache size in the FhirModelBindingOptions object for this purpose so I am removing all this unnecessary code.

Reply:
I like that even better.